src/prometheus-stack/chart/templates/uds-exemption.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: uds.dev/v1alpha1
+kind: Exemption
+metadata:
+  name: monitoring
+  namespace: uds-policy-exemptions
+spec:
+  exemptions:
+    - policies:
+        - Restrict_HostPath_Write
+        - Restrict_Volume_Types
+      matcher:
+        namespace: monitoring
+        name: "^prometheus-kube-prometheus-stack-prometheus-.*"
+      
+    - policies:
+        - Restrict_Volume_Types
+      matcher:
+        namespace: monitoring
+        name: "^kube-prometheus-stack-prometheus-node-exporter-.*"
+
+
```

Comment:
Noticed that yamllint is not being run on these files. It looks like we can make yamllint sort of work with helm by doing these changes:
- add `'!**/chart/templates/uds-*.yaml'` to the ignore file list, after `'**/chart/templates**'`
- change `braces` to disable

With those changes `uds run lint-check` shows a couple issues with the new exemption files as well as one issue with a package file.